pallets/subtensor/src/uids.rs

```diff
@@ -9,6 +9,15 @@ impl<T: Config> Pallet<T> {
         SubnetworkN::<T>::get(netuid)
     }
 
+    /// Resets the trust, emission, consensus, incentive, dividends of the neuron to default
+    pub fn clear_neuron(netuid: u16, neuron_uid: u16) {
+        Emission::<T>::mutate(netuid, |v| v[neuron_uid as usize] = 0);
```

Comment:
can you add the length check before index with neuron_uid.

Reply:
added